plugins/admin/pkg/command/registry/remove.go

```diff
@@ -0,0 +1,153 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/client-contrib/plugins/admin/pkg"
+
+	"github.com/spf13/cobra"
+)
+
+var username string
+var server string
+
+// NewRegistryRmCommand represents the remove command
+func NewRegistryRmCommand(p *pkg.AdminParams) *cobra.Command {
+	var registryRmCmd = &cobra.Command{
+		Use:   "remove",
+		Short: "Remove registry settings",
+		Long:  `Remove registry settings by server and username to delete secret and update ServiceAccount`,
+		Example: `
+# remove registry settings
```

Comment:
Need two chars for indentation, just as kn client did, thanks!

Reply:
Fixed, thanks